.github/workflows/merge.yaml

```diff
@@ -2,29 +2,29 @@ name: Unikorn Merge
 on:
   push:
     branches:
-    - main
+      - main
```

Comment:
Don't reformat things.  It makes reviewing changes particularly hard.  Also if you wish to enforce coding standards on things, then I'd expect an enforcement action, and for said strategy to be applied to every repository for consistency.

Reply:
It's so much more readable 😢